python/gtsam/utils/plot.py

```diff
@@ -108,6 +108,66 @@ def plot_covariance_ellipse_3d(axes,
     axes.plot_surface(x, y, z, alpha=alpha, cmap='hot')
 
 
+def plot_point2_on_axes(axes,
+                        point: Point2,
+                        linespec: str,
+                        P: Optional[np.ndarray] = None) -> None:
+    """
+    Plot a 2D point on given axis `axes` with given `linespec`.
+
+    Args:
+        axes (matplotlib.axes.Axes): Matplotlib axes.
+        point: The point to be plotted.
+        linespec: String representing formatting options for Matplotlib.
+        P: Marginal covariance matrix to plot the uncertainty of the estimation.
+    """
+    axes.plot([point[0]], [point[1]], linespec, marker='.', markersize=10)
+    if P is not None:
+        w, v = np.linalg.eig(P)
+
+        # k = 2.296
```

Comment:
Kill this line? And maybe add a comment on what `k` is?

Reply:
I removed the commented out line and included a comment as to what `k` is (the "sigma" value for the uncertainty ellipse). I specifically chose 5 sigma as that is the same value used in the other functions, such as `plot_pose2_on_axes`.
Thanks!